lib/settings.js

```diff
@@ -48,7 +48,10 @@ function readFrom(envVars) {
         RATE_INTERVAL: 100, // 100ms
         PROCESS_AMQP_DRAIN: true,
         AMQP_PUBLISH_RETRY_DELAY: 100, // 100ms
-        AMQP_PUBLISH_RETRY_ATTEMPTS: 10
+        AMQP_PUBLISH_RETRY_ATTEMPTS: 10,
+        OBJECT_STORAGE_URI: '',
+        OBJECT_STORAGE_TOKEN: '',
+        OBJECT_STORAGE_OUT: false
```

Comment:
My suggestion – is `OBJECT_STORAGE_DISABLED`.

Reply:
`OBJECT_STORAGE_ENABLED` is ok, `OBJECT_STORAGE_DISABLED` is bad.
`OBJECT_STORAGE_DISABLED` will make me change default value to `true`. And when admiral will set `OBJECT_STORAGE_DISABLED` he will pass string `"false"`, which we will need to parse somehow below. I don't want to create a complex check for this.

Reply:
`OBJECT_STORAGE_ENABLED` is better name, than `OBJECT_STORAGE_OUT`